dev-server/templates/client-config.js.mustache

```diff
@@ -2,6 +2,7 @@
   window.hypothesisConfig = function() {
     return {
       // enableExperimentalNewNoteButton: true,
+      enableExperimentalPDFSideBySide: true,
```

Comment:
"Side-by-side" seems reasonable. I'm not sure that it is unambiguously better than "page fit". I don't have a better suggestion, so let's roll with it. If it proves confusing for other people we might need to reconsider. Maybe something to bikeshed on Slack.